docs/monitoring.md

```diff
@@ -1060,10 +1060,10 @@ when running in local mode.
   - hiveClientCalls.count
   - sourceCodeSize (histogram)
 
-- namespace=<Executor Plugin Class Name>
-  - Optional namespace(s). Metrics in this namespace are defined by user-supplied code, and 
-  configured using the Spark executor plugin infrastructure.
-  See also the configuration parameter `spark.executor.plugins`
+- namespace=<Plugin Class Name>
```

Comment:
I think it could make sense to add a similar entry about plugin metrics to the driver componenet instance metrics list, similarly to what has been done for the namespace=JVMCPU, for example.
Please add \< to escape the "<" character (this issue actually comes from SPARK-20891)